pages/properties/nuxtjs.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "NuxtJS",
+    "creator": "GitHub Actions",
+    "description": "Package a NuxtJS site.",
+    "iconName": "nuxtjs",
+    "categories": ["Pages"]
```

Comment:
We should also include language/frameworks targeted by this template in categories. It will be helpful if we use our suggestion engine for Pages in future. Add `nuxtjs` to categories.